ferrischat_webserver/src/guilds/edit_guild.rs

```diff
@@ -13,32 +13,44 @@ pub async fn edit_guild(
     let bigint_guild_id = u128_to_bigdecimal!(guild_id);
 
     let GuildUpdateJson { name } = guild_info.0;
-
-    let db = get_db_or_fail!();
-
-    let resp = sqlx::query!(
-        "UPDATE guilds SET name = $1 WHERE id = $2 RETURNING *",
-        name,
+    
+    sqlx::query!(
+        "SELECT * FROM guilds WHERE id = $1",
         bigint_guild_id
     )
     .fetch_optional(db)
     .await;
+    
+    match name {
+        Some(name) => {
+            let db = get_db_or_fail!();
+
+            let resp = sqlx::query!(
+                "UPDATE guilds SET name = $1 WHERE id = $2 RETURNING *",
```

Comment:
Don't have it return anything. Like I said on Discord, do a separate query to fetch the object after everything is done.